plugin/ochttp/stats.go

```diff
@@ -152,7 +180,8 @@ var (
 	}
 )
 
-// DefaultClientViews are the default client views provided by this package.
+// DefaultClientViews are the default client views provided by this package,
+// provided for easy mass-registration.
```

Comment:
This is odd wording "provided for easy mass-registration".

I'd rather say "DefaultClientViews are the default client views to register for HTTP clients."